File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/inventory/InventoryGUICreatorHub.java
package com.gmail.borlandlp.minigamesdtools.gui.inventory;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class InventoryGUICreatorHub extends CreatorHub {
    public DrawableInventory createInventory(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (DrawableInventory) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/inventory/InventoryGUICreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.gui.inventory

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class InventoryGUICreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): DrawableInventory {
        return super.create(itemID, dataProvider) as DrawableInventory
    }
}